src/Infrastructure/Services/AppUserService.cs

```diff
@@ -23,10 +25,11 @@ public async Task<Guid> AddAsync(AddAppUserDTO appUserDTO)
             return appUser.Id;
         }
 
-        public async Task<IEnumerable<GetAppUserDTO>> GetAllAsync()
+        public async Task<PagedList<GetAppUserDTO>> GetPagedAsync(PagedQueryBase query)
         {
-            var appUsers = await _unitOfWork.AppUsers.GetAllAsync();
-            return _mapper.Map<IEnumerable<GetAppUserDTO>>(appUsers);
+            var appUsers = await _unitOfWork.AppUsers.GetPagedAsync(query);
+            var appUsersDTO = new PagedList<GetAppUserDTO>(_mapper.Map<List<GetAppUserDTO>>(appUsers), appUsers.TotalCount, appUsers.CurrentPage, appUsers.PageSize);
```

Comment:
in order to make code more readable create another var and assing this "_mapper.Map<List<GetAppUserDTO>>(appUsers)" to it.

This recommendation applies to other identical sections of code as well.

Reply:
done